aspen/__init__.py

```diff
@@ -70,27 +70,15 @@
 from .http.response import Response
 from . import json_ as json
 from .logging import log, log_dammit
+from .simplates.renderers import BUILTIN_RENDERERS, RENDERERS
```

Comment:
Remind me again why we need these in here? Is this part of backcompat for plugins, or ... ?